### PR TITLE
Created Bug_report template

### PR DESCRIPTION
**Pull request by M-casado**
Created bug issue template for PyEGA3 GH repository - Jira ticket: EE-1778

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+**Description of the bug**
+
+A clear and concise description of what the bug is. 
+
+*(e.g. Used option `--delete-temp-files` did not delete all temporary files after failed transfer)*
+
+**Used versions** (please complete the following information)
+ - Operating System version: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*(e.g. `Windows 10.0.19041`)*
+ - Python version: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*(e.g `3.7.0`)*
+ - PyEGA3 version: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; *(e.g `EGA python client version 3.4.0`)*
```
Comment (malloryfreeberg) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: The `&nbsp;` non-breaking spaces look weird here. Should these just be regular spaces?

**Comment by malloryfreeberg** [comment]
LGTM except for one minor formatting comment. Please have at least 1 other person review.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (M-casado) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: If I'm not mistaken, GitHub's markdown do not allow for tabs within text lines. To make it look nicer I chose to use `&nbsp;`, but it's completely non-essential.

Comment (malloryfreeberg) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: I'm fine with the `&nbsp` if the template renders more nicely this way.

**Review by malloryfreeberg — APPROVED** [review]
LGTM

**Review by malloryfreeberg — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (afoix) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: Well done on include the basic requirements.

**Review by afoix — APPROVED** [review]
Looks good to me!

**Pull request merged**